server/client.go

```diff
@@ -735,6 +759,7 @@ func (c *client) readLoop() {
 	nc := c.nc
 	s := c.srv
 	c.in.rsz = startBufSize
+	c.in.losc = time.Now().UnixNano()
```

Comment:
Can be second resolution yes? We would not do sub-second would we?

Reply:
In tests.. using 1sec is likely to cause to wait closer than 2 sec instead.. will make the test that check this one a bit longer..

Reply:
ok, maybe just do time.Time here.